site/public/sitemap.xml

```diff
@@ -34,5 +34,25 @@
 		<changefreq>weekly</changefreq>
 		<priority>0.5</priority>
 	</url>
+	<url>
+		<loc>/salt/components/button/accessibility.mdx</loc>
```

Comment:
Just noticed that this whole `sitemap.xml` file seems to be broken. The contents of `<loc>` should be the the full URL to the page (see https://www.sitemaps.org/protocol.html), but instead seems to be the path to the source file.

I realise that this bug has nothing to do with this PR, so I've raised a separate issue for it: #1559